### PR TITLE
feat: remove blockd

### PR DESCRIPTION
**Pull request by andrewrynhard**
Signed-off-by: Andrew Rynhard <andrew@andrewrynhard.com>

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ steps:
   environment:
     BUILDKIT_HOST: tcp://buildkitd.ci.svc:1234
   commands:
-    - make
+    - make -j
```
Comment (smira) [inline .drone.yml]: is buildkit able to track deps for concurrent builds correctly?

Comment (andrewrynhard) [reply .drone.yml]: That is what they claim.

Comment (andrewrynhard) [reply .drone.yml]: That is what they claim.

Comment (andrewrynhard) [reply .drone.yml]: This was only for testing. I have removed this from this PR.

**Pull request merged**